Source/Scene/WebMapServiceImageryProvider.js

```diff
@@ -268,6 +273,7 @@ function WebMapServiceImageryProvider(options) {
   pickFeatureResource.setQueryParameters(pickFeatureParams, true);
 
   this._resource = resource;
+
```

Comment:
Unneeded extra line here

Reply:
Hi @ebogo1 , I removed the extra line in this commit: https://github.com/CesiumGS/cesium/pull/9563/commits/addacfe22e476625765ca9af5ac68cdcb86b033e
Thank you.